PWA.md

```diff
@@ -0,0 +1,117 @@
+# PWA
+
+Elk provides a PWA (Progressive Web App) that can be installed on your desktop/device. This allows you to use Elk as a native app on your device, and it will work offline.
+
+The main goal of the PWA is to provide a native app experience on mobile devices with Web Push Notifications and Web Share Target API capabilities.
+
+Web Share Target will allow you to share content from other apps to Elk in mobile browsers.
+
+## Mobile Support
+
+Some mobile browsers will allow you to install the PWA as a native app, and some others will only allow you to add the PWA to your home screen.
+
+If you're using a mobile browser that supports the PWA installation, you will see a banner at the bottom of the screen with the option to install the PWA.
+
+If the browser also supports `beforeinstallprompt` event, ElK will show a prompt (on the right aside on wide screens, or at top on small screens) to allow you to install the PWA directly.
+
+If the browser doesn't support the PWA installation, check following entries:
+
+### Safari iOS
+
+Right now, you cannot use Web Push Notifications on Safari mobile browsers, since it doesn't support the Web Push API yet at operating system level.
+
+You can check the status of the Push API on Safari mobile browsers on [this page](https://caniuse.com/notifications).
+
+Visit also [this site](https://firt.dev/notes/pwa-ios/) to check the status of PWA Capabilities on Safari mobile browsers.
+
+To install a Progressive Web App (PWA) on Safari, follow these steps:
+- Tap the "Share" icon at the bottom of the screen.
+- Scroll down and tap "Add to Home Screen".
+- Customize the name of the app (if desired) and tap "Add".
+- The PWA should now appear on your home screen.
+
+### Firefox
+
+To install a Progressive Web App (PWA) on Firefox, follow these steps:
+- Tap the menu button (three horizontal lines) at the bottom of the screen.
+- Tap "Page options" and then "Install website as app".
```

Comment:
No Page options on my Android firefox, directly the Install action button:
![Screenshot_2023-02-14-14-27-54-63_3aea4af51f236e4932235fdada7d1643.jpg](https://user-images.githubusercontent.com/2922851/218752584-abd536a0-e466-4e18-8724-3f6959ec589c.jpg)